src/components/dropdown/Dropdown.tsx

```diff
@@ -236,7 +236,7 @@ export class Dropdown<T = LabelValue> extends React.Component<
 
   handleIsOptionSelected = this.props.isOptionSelected
     ? (option: any) => this.props.isOptionSelected(option.data)
-    : undefined;
+    :  (option: DropdownOption<T>, selectValue: any) => selectValue?.some(i => i === option);
```

Comment:
could we give a better type to selectValue? it'll be easier to maintain
Also I see you use some on it, are we sure it's always an array or may happen it's a plain object?

Reply:
Type updated. 
I just tested and selectedValue is always an array.